Framework/CurveFitting/src/Algorithms/CalculateChiSquared.cpp

```diff
@@ -250,7 +250,7 @@ class ChiSlice {
                               bool &ok) {
 
     auto base = ChebfunBase::bestFitAnyTolerance(lBound, rBound, *this, P, A,
-                                                 1.0, 1e-4, 129);
+                                                 1e-4, 129, 1.0);
```

Comment:
Could you help me understand why this change is required?

The definition of `bestFitAnyTolerance` says that the last 3 parameters are `maxA`, `tolerance` & `maxSize` so it would imply that the tolerance has changed to `129`? Is that correct?

Reply:
I'd have to defer to @alicerussell1 , as this is a section of Roman's PR that I've inherited but not touched

Reply:
I can't see a reason for this, I think it might be a mistake that should probably be changed back

Reply:
Fixed in #26815